### PR TITLE
v1.0.0 Polish

### PR DESCRIPTION
**Pull request by locks**
- [x] Prepare addon

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ember/string",
-  "version": "0.0.0",
+  "version": "1.0.0",
```
Comment (rwjblue) [inline package.json]: I generally only update this during publishing. Nothing wrong with doing it this way though...

**Comment by locks** [comment]
Waiting on https://github.com/babel/ember-cli-babel/pull/260 being released.

**Comment by rwjblue** [comment]
Was released, probably should update to latest (7.4.0 ATM).

**Comment by locks** [comment]
@rwjblue done 😁 was waiting on CI to confirm.

**Pull request merged**